### DISCUSSION TITLE
Using matchmaker exchange for local installs

### DISCUSSION THREAD

**Discussion by anjalijain22** [broadinstitute/seqr#3626] [Q&A] [answered]
Hi, hope you're well!

I'm curious to learn how seqr's MME functionality works for local installs. I understand that Broad's seqr instance as well as BCH seqr are nodes of matchmaker exchange but how would that work if I wanted to submit a request from my local instance? Can you point me to any relevant docs on this?

Many thanks!

**Comment by hanars** [comment, marked as answer]
While the code for connecting to the matchmaker API is fully available in the seqr local install, in order to connect you must go through the process outlined by the matchmaker exchange to become a node and receive authorization to connect with each other node. The seqr team is not involved in this process, and unfortunately has no way to help expedite it. The matchmaker web page includes information about what the requirements are and who to contact about connecting: https://www.matchmakerexchange.org/i_have_a_database.html

**Reply by anjalijain22** [reply]
That makes sense, thanks for sharing this info!
